### ISSUE TITLE
Be explicit about fallback roles being unsupported

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1916]
Currently, when you do something like this:

```html
<div role="application tablist">
```

Axe-core reports "ARIA role is valid". That's a confusing message. Instead it should say something like "Use only one role value, since fallback roles are not widely supported".

## Proposal on how to do this

1. support fallback roles in the existing *role tools, have them look at each role in the role="" attribute
2. Add a new check that looks just for use of fallback roles

That way something like `role="foobar textbox select"` would be reported as:

```
Fix all of the following:
- Use only one role value, since fallback roles are not widely supported"
- Abstract role 'select' cannot be directly used
- Role 'foobar' must be a valid ARIA roles
```

We'd have to update the message to indicate which role is invalid / abstract, and report multiples. (e.g. "abstract roles 'select', and 'landmark' cannot be directly used".

**Comment by jnurthen** [comment]
Fallback roles are pretty well supported except on IE. For those of us writing products which don't support IE is there any chance we could have a way of allowing fallback roles.
I'm trying to write a meter right now and am using `role="meter progressbar"` causing aXe errors

**Comment by WilcoFiers** [comment]
Getting axe-core to run entirely with support for fallback roles is going to take a while. But I'm pretty confident some time next year this will change. This change will be the first step, by creating a separate check for this, it also makes it possible to turn the check off, separate from the rest of the rule.

I also think it's still worth pointing out that IE still makes up 15% of the WebAIM survey market. That's a significant enough portion that I'm comfortable saying we should flag it for a while still.

**Comment by jeankaplansky** [comment]
Release notes updated 2/04/2020

**Comment by padmavemulapati** [comment]
Instead Aria-role is valid, if displayed some non-confusing accessible name could be better , verified with aria-role.html rule test file, 
![image](https://user-images.githubusercontent.com/47409579/74037502-6c813680-49e4-11ea-888d-2f9c1b66ec2f.png)
